tests/torch/test_compile_torch.py

```diff
@@ -146,15 +147,27 @@ def compile_and_test_torch_or_onnx(  # pylint: disable=too-many-locals, too-many
     else:
         inputset = (numpy.random.uniform(-100, 100, size=(n_examples, *to_tuple(input_shape))),)
 
-    # FHE vs Quantized are not done in the test anymore (see issue #177)
-    if not simulate:
+    # Compile our network with the same bitwidth in simulation and FHE
+    if qat_bits == 0:
```

Comment:
are we really testing with qat_bits = 0 ?! 
For which case is it ?
Can you add a comment, to explain it, please.

Reply:
qat_bits==0 simply means use PTQ